runtime/service_integ_test.go

```diff
@@ -1945,8 +1945,8 @@ func TestOOM_Isolated(t *testing.T) {
 		containerd.WithSnapshotter(defaultSnapshotterName),
 		containerd.WithNewSnapshot("snapshot-"+vmID, image),
 		containerd.WithNewSpec(
-			// The container is having 2MB of memory.
-			oci.WithMemoryLimit(2*1024*1024),
+			// The container is having 3MB of memory.
```

Comment:
Can you cut a GitHub issue? A bit concerning to see we have less memory somehow.

Reply:
Sure. so the issue should be cut to firecracker or firecracker-containerd?

Reply:
Cut issue #572